mrmap/quality/tables.py

```diff
@@ -21,5 +29,50 @@ class Meta:
         fields = ('metadata', 'config', 'passed', 'report', 'created_at', 'created_by_user', 'owned_by_org')
         prefix = 'conformity_check_run-table'
 
-    def render_report(self, record):
+    @staticmethod
+    def render_report(record):
         return Link(url=record.get_report_url(), content='Validation report').render(safe=True)
+
+
+class ConformityCheckRunExtraFieldsTable(tables.Table):
+    latest_check = tables.Column(verbose_name=_('Latest check'),
+                                 attrs={"th": {"class": "col-sm-1"}},
+                                 empty_values=[])
+
+    total_check_count = tables.Column(verbose_name=_('Total check count'),
+                                      attrs={"th": {"class": "col-sm-1"}},
+                                      empty_values=[])
+
+    class Meta:
+        model = ConformityCheckRun
+        fields = ["latest_check", "total_check_count"]
+        prefix = "conformity_check_run-extra-fields-table"
+
+    @staticmethod
+    def render_latest_check(record):
+        latest_check = ConformityCheckRun.objects.get_latest_check(record)
+        if latest_check:
+            latest_check_creation_datetime_str = latest_check.created_at.strftime("%m/%d/%Y %I:%M%p")
+            latest_check_passed = latest_check.passed
+            if latest_check_passed is True:
+                icon = get_icon(IconEnum.OK, 'text-success')
+            elif latest_check_passed is False:
+                icon = get_icon(IconEnum.NOK, 'text-danger')
+            else:
+                icon = get_icon(IconEnum.PENDING, 'text-warning')
+
+            link_to_report = reverse('quality:conformity_check_run_report', kwargs={'pk': latest_check.pk})
+            content = f'<span>{latest_check_creation_datetime_str} {icon}</span>'
+            return Link(url=link_to_report, content=content).render(safe=True)
+        else:
+            return None
+
+    @staticmethod
+    def render_total_check_count(record):
+        total_check_count = ConformityCheckRun.objects.get_total_check_count(record)
```

Comment:
Same as above. Use again annotations to avoid from sql query's for every rows/column. In your implementation you got n+2 query's minimum to get all table data.